mne_bids/dig.py

```diff
@@ -543,6 +546,116 @@ def _read_dig_bids(electrodes_fpath, coordsystem_fpath,
     # put back in unknown for unknown coordinate frame
     if coord_frame == 'unknown':
         for ch in raw.info['chs']:
-            ch['coord_frame'] = _str_to_frame['unknown']
+            ch['coord_frame'] = MNE_STR_TO_FRAME['unknown']
         for d in raw.info['dig']:
-            d['coord_frame'] = _str_to_frame['unknown']
+            d['coord_frame'] = MNE_STR_TO_FRAME['unknown']
+
+
+# Remove once we depend on MNE-Python 1.0+
+def _get_montage(info):
+    try:
+        info.get_montage
+    except AttributeError:
+        return mne.io.RawArray(np.zeros((info['nchan'], 1)),
+                               info).get_montage()
+    else:
+        return info.get_montage()
+
+
+@verbose
+def template_to_head(info, space, coord_frame='auto', unit='auto',
+                     verbose=None):
+    """Transform a BIDS standard template montage to the head coordinate frame.
+
+    Parameters
+    ----------
+    %(info_not_none)s The info is modified in place.
+    space : str
+        The name of the BIDS standard template. See
+        https://bids-specification.readthedocs.io/en/stable/99-appendices/08-coordinate-systems.html#standard-template-identifiers
+        for a list of acceptable spaces.
+    coord_frame : str
+        BIDS template coordinate systems do not specify a coordinate frame
+        so this must be determined by inspecting the documentation for the
+        dataset or the electrodes.tsv file. Must be ``mri``, ``mri_voxel``
+        or ``ras``. If ``auto``, the montage will be assumed to be in
+        ``mri_voxel`` if the coordinates are strictly positive and
+        ``scanner RAS`` otherwise.
+
+        .. warning::
+
+            ``scanner RAS`` and ``surface RAS`` coordinates frames are similar
+            so be very careful not to assume a BIDS dataset's coordinates are
+            in one when they are actually in the other. The only way to tell
+            for template coordinate systems, currently, is if it is specified
+            in the dataset documentation.
+
+    unit : str
+        The unit that was used in the coordinate system specification; either
+        ``m`` or ``mm``. If ``auto``, ``m`` will be inferred if the montage
+        spans less than -1 to 1 in units (meters) and ``mm`` otherwise. If the
+        ``coord_frame`` is 'mri_voxel', ``unit`` will be ignored.
```

Comment:
```suggestion
    unit : 'm' | 'mm' | 'auto'
        The unit that was used in the coordinate system specification.
        If ``'auto'``, ``'m'`` will be inferred if the montage
        spans less than ``-1`` to ``1`` in units, and ``'mm'`` otherwise. If the
        ``coord_frame`` is ``'mri_voxel'``, ``unit`` will be ignored.
```